tutorial-template.asciidoc

```diff
@@ -0,0 +1,96 @@
+This template can be used for creating tutorials within the Observability documentation.
+This should give you a format to follow and make it easier for you to get started writing
+your tutorial. You can use this asciidoc template as a framework, replacing the text with your own. 
+
+// The title of your tutorial should focus on what the reader will achieve.
+// For example, How to ingest custom data into Elasticsearch.
+// The anchor id name, in this case [[tutorial-title]], must be unique and can only be used once in a doc. You can link to any ID within a document. For example,
+// if you need to link to this section from elsewhere, you would use <<tutorial-title>> as the reference. 
+[[tutorial-title]]
+= Tutorial title
+
+Introduce the reader to what they will do in this tutorial and provide a list
+of what the reader will learn. If required, include links to topics within the Observability docs. 
+
+When you have finished this tutorial, you will be familiar with:
+
+// This is an ordered list and each item is using a shared attribute for the
+// product name. Shared attributes can be found here: https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc
+. Installing {es}
+. Installing {kib}
+. Installing {filebeat}
+
+
+// This float marker is placed before an anchor id so that each section in this file remains on the same page when converted to HTML.
+[float]
+[[what-you-need]]
+// The section heading. 
+== What you need
+
+Use this section to tell the reader what they will need, or any prerequisite knowledge
+they require to complete the tutorial.
+
+
+[float]
+[[the-first-step]]
+== Step 1: The first step
+
+Break your tutorial down into discrete steps that the reader can follow. A task should be broken 
+down into as many steps as are needed.
+
+To highlight a command within a sentence, use this format: `sample command`.
+
+To include code or command blocks, use a code block. Specify the source language, as we use the language 
+as a hint for the syntax highlighter. For example; 
+
+[source,shell]
+----
+./elastic-agent -c elastic-agent-standalone.yml run
+----
+
+To include a sample file, specify the source format. For example:
+
+[source,json]
+{
+    "sample_id": ENT,
+    "sample_name": "String",
+    "sample_number": INT,
+    "line_number": "String",
+    "speaker": "String",
+    "text_entry": "String",
+}
+
+To include an image, save it in a folder in the Observability-docs repo, and
+include it using a path relative to the document where the `image::` statement appears. For example, 
```

Comment:
I would say "by using an `image::` statement" (then show the example).

We need to revisit how we specify the image path to make it possible to move things around more easily, but that's somewhat more complicated in asdciidoc than it probably should be. :-) Probably best right now to be less precise.